src/xb-machine.c

```diff
@@ -1490,6 +1490,19 @@ xb_machine_func_ne_cb (XbMachine *self,
 					   error);
 	}
 
+	/* TEXT:INTE */
+	if (xb_opcode_cmp_val (&op2) && xb_opcode_cmp_str (&op1)) {
```

Comment:
Nitpick: it’s a bit weird that the comment is in op1:op2 order, but the tests in the `if` statement are in `op2 && op1` order.

Same below and above.

Reply:
Not a nitpick, that's perfectly valid. Will fix up.